src/spikeinterface/postprocessing/principal_component.py

```diff
@@ -385,12 +386,15 @@ def _fit_by_channel_local(self, n_jobs, progress_bar):
 
         tmp_folder = p["tmp_folder"]
         if tmp_folder is None:
-            tmp_folder = "tmp"
-        tmp_folder = Path(tmp_folder)
+            if n_jobs > 1:
+                import tempfile
```

Comment:
I guess this could go on top as it is part of the standard library?

Reply:
done